tests/PHPUnit/Unit/Validator/AlphaNumSpacesTest.php

```diff
@@ -0,0 +1,58 @@
+<?php
+/**
+ * Matomo - free/libre analytics platform
+ *
+ * @link https://matomo.org
+ * @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ */
+
+namespace Piwik\Tests\Unit\Translation\Loader;
+
+use PHPUnit\Framework\TestCase;
+use Piwik\Validators\AlphaNumSpaces;
+
+/**
+ * @group Validator
+ * @group AlphaNumSpaces
+ * @group AlphaNumSpacesTest
+ */
+class AlphaNumSpacesTest extends TestCase
+{
+    public function test_validate_success_strings()
+    {
+        $this->validate('onlyalpha');
+        $this->validate(123);
+        $this->validate('abc 123');
+        $this->validate('abc 123 and 0');
```

Comment:
might be good to also test edge cases like German umlauts, some Chinese characters and maybe even some utf8 emojis maybe

Reply:
definitely, I will do that 👍

Reply:
I added more test cases with special characters. Emoji is not supported with this regex. Do you think we should let emojis as well?